src/Entity/Adherent.php

```diff
@@ -75,6 +80,15 @@ class Adherent implements UserInterface, UserEntityInterface, GeoPointInterface,
     use EntityPostAddressTrait;
     use LazyCollectionTrait;
 
+    /**
+     * @ORM\Column(length=25, unique=true, nullable=true)
+     *
+     * @Assert\Length(max=25, groups={"anonymize"})
```

Comment:
Should we add a minimum length or not ?
And a regex pattern ?

what if i choose `a` or `.` as a nickname ? or `'` \o/

Reply:
Done.